pipelines/performance-tests-pipeline.yml

```diff
@@ -37,12 +37,13 @@ resources:
     password: ((gcp.service_account_json))
 
 - name: census-rm-terraform
-  type: git
+  type: github-release
   source:
-    branch: perf-resource-workaround
-    uri: git@github.com:ONSdigital/census-rm-terraform.git
-    private_key: ((github.service_account_private_key))
-    
+    owner: ONSdigital
+    repository: census-rm-terraform
+    access_token: ((github.access_token))
+    branch: perf-resource-workaround # TO DO: use github release resource when this temporary branch is finished (remove branch option)
```

Comment:
This one will need to stay as a `git` resource if we want to keep it working off the branch for now.

Reply:
I have just reverted this back to what it was in master...can you check it over please?